### ISSUE TITLE
Any progress on google_inception_v3 re-training?

### ISSUE THREAD

**Issue by wangxianliang** [smichalowski/google_inception_v3_for_caffe#5]
Hi, what is the final accuracy on you re-trained google_inception_v3 model?
Thanks!

**Comment by smichalowski** [comment]
still working on it ...

```
I0615 08:00:07.854004  1610 solver.cpp:337] Iteration 4000, Testing net (#0)
I0615 08:00:15.725909  1610 solver.cpp:404]     Test net output #0: acc/top-1 = 0.628
I0615 08:00:15.725968  1610 solver.cpp:404]     Test net output #1: acc/top-5 = 0.996
I0615 08:00:15.725986  1610 solver.cpp:404]     Test net output #2: loss = 1.68205 (* 1 = 1.68205 loss)
```

best which I achieved trained on tiny imagenet set (5 categories) to speed things up.

**Comment by n3011** [comment]
@wangxianliang a better implementation https://github.com/n3011/Inception_v3_GoogLeNet

**Comment by smichalowski** [comment]
just updated repo with new train_val.prototxt that achieves expected results.

**Comment by wangxianliang** [comment]
Is the trained result only a subset of ImageNet?
